assets/sass/_container.scss

```diff
@@ -0,0 +1,52 @@
+// scss-lint:disable IdSelector
+#okta-sign-in {
+// scss-lint:enable IdSelector
+
+  // Html and body styles in _base.scss that were moved here
+  font-family: $fonts;
+  font-size: $font-size;
+  font-weight: $font-weight-body;
+  line-height: 1.4;
+  background-color: $secondary-bg-color;
```

Comment:
- I think originally it came from the page background. `background-color` can be removed
- `min-height` can be removed. What's odd is that this renders normally.
- `position` and `overflow` can be removed